src/procedures/procedures.11tydata.yaml

```diff
@@ -0,0 +1,140 @@
+---
+procedureNewZigbee:
+  title: Setting up a new Zigbee network using SkyConnect
+  steps:
+    - title: Plug in the extension cable
+      image: skyconnect-raspi-01.jpg
```

Comment:
Your photos are all quite big, 180kB. They are going to be shown in a quite small space. Consider reducing their size by uploading them to https://tinypng.com/

Reply:
Thank you! Done.